src/node_jsvmapi.cc

```diff
@@ -23,6 +23,24 @@
 #include <vector>
 #include <string.h>
 
+static v8::Persistent<v8::Value> *lastException = 0;
```

Comment:
Not if it's initially empty. The isolate can be provided later in the call to `Reset()`.